source/aws-bootstrap-kit/lib/account-handler/index.ts

```diff
@@ -40,26 +40,19 @@ export async function onEventHandler(
     case "Create":
       const awsOrganizationsClient = new Organizations({region: 'us-east-1'});
       try {
-        const tags: { Key: string; Value: any; }[] = [];
-        Object.keys(event.ResourceProperties).forEach( propertyKey => {
-          if( propertyKey != 'ServiceToken' ) tags.push({Key: propertyKey, Value: event.ResourceProperties[propertyKey]});
-        });
         const data = await awsOrganizationsClient
         .createAccount({
           Email: event.ResourceProperties.Email,
-          AccountName: event.ResourceProperties.AccountName,
-          Tags: tags
+          AccountName: event.ResourceProperties.AccountName
         })
         .promise();
         console.log("create account: %j", data);
         return { PhysicalResourceId: data.CreateAccountStatus?.Id };
       } catch (error) {
         throw new Error(`Failed to create account: ${error}`);
       }
-    case "Update":
+    default: // just return the resource (we cannot update or delete an account)
```

Comment:
We need to document that. We cannot remove accounts. 
Accounts can just be detach from the organization but in order to do so, you need to configure billing and a credit card..., definitely not automatable.
Also we cannot throw an exception because it fails the stack deletion (or fails a rollback when there are issues elsewhere)